plugins/ipam/static/README.md

```diff
@@ -60,3 +60,9 @@ The plugin also support following [capability argument](https://github.com/conta
 The following [args conventions](https://github.com/containernetworking/cni/blob/master/CONVENTIONS.md#args-in-network-config) are supported:
 
 * `ips` (array of strings): A list of custom IPs to attempt to allocate, with prefix (e.g. '10.10.0.1/24')
+
+Note: If some of above is used at same time, one input sources are used as following priority:
```

Comment:
nit:  `Notice: If some of above are used at same time, only one will work according to the priorities below`

Reply:
fixed.